src/styles/main.scss

```diff
@@ -1,3 +1,6 @@
-body {
-  margin: 0;
```

Comment:
Also in check.list write that color in phone, email etc. should change after hover

![image](https://github.com/mate-academy/layout_landing-page/assets/142230575/41f59b91-85f7-4e4f-8d43-c8f08e27127b)